src/System.Data.SqlClient/tests/ManualTests/SQL/IntegratedAuthenticationTest/IntegratedAuthenticationTest.cs

```diff
@@ -15,7 +15,9 @@ namespace System.Data.SqlClient.ManualTesting.Tests
 {
     public static class IntegratedAuthenticationTest
     {
-        [CheckConnStrSetupFact]
+        private static bool IsIntegratedSecurityEnvironmentSet() => DataTestUtility.IsIntegratedSecuritySetup();
+
+        [ConditionalFact(nameof(IsIntegratedSecurityEnvironmentSet))]
```

Comment:
I think it would be good to check for whether Connection Strings have been set in the env AND if Integrated security variable is set.

Reply:
Unfortunately, using both the facts together throw an exception as seen below, since both `CheckConnStrSetupFact` and `ConditionalFact` derive from `FactAttribute` it seems like we can only use either of the two, hence I chose to use ConditionalFact, similar to how `IsUsingManagedSNI` and `IsLocalDBEnvironmentSet` are handled.
```cs
System.InvalidOperationException : Test method 'System.Data.SqlClient.ManualTesting.Tests.IntegratedAuthenti
  cationTest.IntegratedAuthenticationTestWithConnectionPooling' has multiple [Fact]-derived attributes
```

Here are my suggestions to check for both of them, kindly let me know your thoughts.
- Checking ConnectionString Setup implicitly with `ConditionalFact` ?
OR
- Define IsConnectionStringConfigured in similarly as seen below in DiagnosticTest and check for 2 conditions
`[ConditionalFact(nameof(IsIntegratedSecurityEnvironmentSet), nameof(IsConnectionStringConfigured))]`
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs#L26
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs#L206

Reply:
> Define IsConnectionStringConfigured in similarly as seen below in DiagnosticTest and check for 2 conditions
[ConditionalFact(nameof(IsIntegratedSecurityEnvironmentSet), nameof(IsConnectionStringConfigured))]

I like this better, it specifies that the test is dependant on two pre-conditions in the attribute itself instead of jumping into another function and getting to know that there are a couple of Environment variables to be set.

Reply:
Done, also updated `LocalDBTest.cs` file to check both conditions, where earlier only `ConditionalFact(nameof(IsLocalDBEnvironmentSet)` was being checked.

Reply:
You don't need to check for local DB connection strings. Those tests have connection strings embedded